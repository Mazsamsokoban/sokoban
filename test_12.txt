loadtest 12
showstate w
showstate box
showmaze
showstate w
showstate box