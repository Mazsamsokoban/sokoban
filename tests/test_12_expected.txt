f1:w bf:box f2:null 

name:w
field:f1
direction:Neutral
points:1
pushforce:10.0

name:box
field:bf
direction:Neutral
owner:w
pusher:null

f1:w bf:box f2:null 

name:w
field:f1
direction:Neutral
points:1
pushforce:10.0

name:box
field:bf
direction:Neutral
owner:w
pusher:null

